cookbooks/swift/files/default/etc/systemd/system/prometheus.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Prometheus Server
+After=multi-user.target
+
+[Service]
+ExecStart=/usr/local/bin/prometheus \
+  --config.file="/etc/prometheus/prometheus.yaml" \
+  --storage.tsdb.path="/tmp/prom/data/" \
+  --storage.tsdb.retention.time=1d
```

Comment:
"One day" seemed like a long enough time to be useful, but not so long as to prove onerous. Might warrant more thought.

FWIW, after ~2h up (including running probe tests), `/tmp/prom` is sitting at ~15MB.